released/packages/coq-msets-extra/coq-msets-extra.1.2.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "thomas@tuerk-brechen.de"
+homepage: "https://github.com/thtuerk/MSetsExtra"
+dev-repo: "git+https://github.com/thtuerk/MSetsExtra.git"
+bug-reports: "https://github.com/thtuerk/MSetsExtra/issues"
+authors: ["FireEye Formal Methods Team <formal-methods@fireeye.com>" "Thomas Tuerk <thomas@tuerk-brechen.de>"]
+license: "LGPL 2.1"
```

Comment:
```suggestion
license: "LGPL- 2.1-only"
```
We are actively pushing for using [SPDX](https://spdx.org) license identifiers. Can you confirm that this is LGPL 2.1 only, and not [LGPL 2.1 or later](https://spdx.org/licenses/LGPL-2.1-or-later.html)?